python/taichi/main.py

```diff
@@ -124,31 +124,38 @@ def _get_available_examples() -> set:
         examples_dir = TaichiMain._get_examples_dir()
         all_examples = examples_dir.rglob('*.py')
         all_example_names = {
-            str(f.resolve()).split('/')[-1].split('.')[0]
+            Path(f).stem
```

Comment:
I hope this fixes #1099 now. It was a stupid mistake I made to rely on slash splits instead of `Path` objects 🤦  I haven't set up a Windows dev/test env yet, so this is still not 100% sure

Reply:
Also note this will likely not work for examples that are named as `mpm88.v2.py`, I just hope we never follow that pattern over `mpm88_v2.py` 🤔